pandas/tests/computation/test_eval.py

```diff
@@ -860,19 +860,6 @@ def test_basic_frame_series_alignment(
     def test_basic_series_frame_alignment(
         self, request, engine, parser, index_name, r_idx_type, c_idx_type
     ):
-        if (
```

Comment:
Maybe fixing a random seed might help make this more deterministic?

Reply:
i think that would work, but in both this case and the JSONArray case there may be an actual bug in play, so we would want to have an xfailed test?

Reply:
Fair point. From https://github.com/pandas-dev/pandas/pull/46796 for example, probably best if its xfailed even if strict=False due to some seemingly random sorting bug